src/components/Charts/utils.ts

```diff
@@ -190,18 +190,8 @@ export const getXTickFormat = (
   }
 };
 
-/**
- * For mobile, we only show every other month
- */
-function getMobileDateTicks(tickValues: Date[]): Date[] {
-  return tickValues.filter((value: Date, i: number) => i % 2 === 0);
-}
-
 export function getFinalTicks(isMobile: boolean, ticks: Date[]): Date[] {
-  if (isMobile) {
-    return getMobileDateTicks(ticks);
-  }
-  return ticks;
+  return ticks.filter((value: Date, i: number) => i % (isMobile ? 3 : 2) === 0);
```

Comment:
At this point, even displaying a tick every other month on mobile is still making the labels squish together. So I adjusted it to be every quarter on mobile. On desktop however, I don't think we need to split them by quarter yet (doing so makes the x-axis labels appear pretty bare). So I adjusted it to every 2 months.